SVGEngine.mm

```diff
@@ -103,12 +103,20 @@ @interface SVGAttributeSet () {
                                               valueOptions:NSMapTableStrongMemory];
     NSMutableArray * const paths = [NSMutableArray new];
 
+    unsigned int depthWithinUnknownElement = 0;
+
     while(xmlTextReaderRead(_xmlReader) == 1) {
         int const type = xmlTextReaderNodeType(_xmlReader);
         const char * const tag = (char *)xmlTextReaderConstName(_xmlReader);
         
         CGPathRef path = NULL;
-        if(type == XML_READER_TYPE_ELEMENT && strcasecmp(tag, "path") == 0)
+        if(type == XML_READER_TYPE_ELEMENT && strcasecmp(tag, "svg") == 0) {
```

Comment:
Let's insert a comment here along the lines of `// Prevent <svg> from being ignored`

Reply:
👍